src/lib/data.ts

```diff
@@ -43,9 +46,39 @@ export function setWithoutEmit(key: string, value: any | undefined): any {
     return value;
 }
 
-export function set(key: string, value: any | undefined): any {
+export function set(key: string, value: any | undefined, persist: boolean = false): any {
```

Comment:
How about white-listing what key's should persist, that way we don't have to remember in code which keys we should be passing `true` here to? So instead of checking if `persist === true`, we check if the `key` is in the set of keys we persist?

Reply:
Yeah - I was initially thinking that this would go in `preferences.defaults`  

But of course preferences aren't the only kind of data that gets stored.

So I copped out of making another list.

But I think you are right.